tests/test_signing.py

```diff
@@ -110,15 +111,34 @@ def test_verify(self):
 
         secret = u'secret'  # unicode is what we usually get from api/db..
 
-        verified = signing.verify(
+        signatures = (
             '346f3d53ad762f3ed3fb7f2427dec2bbfaf0338bb7f91f0460aff15c',
+            u'346f3d53ad762f3ed3fb7f2427dec2bbfaf0338bb7f91f0460aff15c',
+        )
+
+        for signature in signatures:
+            verified = signing.verify(
+                signature,
+                secret,
+                params,
+                url,
+                'POST',
+            )
+            self.assertTrue(verified)
```

Comment:
Maybe we could add a meaningful message here in case of failure? So we will know on which _signature_ verification has failed.